### PR TITLE
Corrected Misleading R-Instat message 3.5.4 to 3.6.0 on installation

### LINKED ISSUE #5374: Misleading R version Message
**Describe the bug**
Wrong R version requirement message. It says it requires 3.5.4 which apparently does not exist.  The message should be r version 3.6.0.

image

**what happened exactly**
I had a problem with my R. When I uninstalled and installed again (version 3.6.0). I kept getting the above error.  But finally found a way out to have R-instat run.

### PR DESCRIPTION
**Pull request by stevenndungu**
fixes #5374
@africanmathsinitiative/developers  this is ready for review

### CODE REVIEW

**Comment by dannyparsons** [comment]
I don't think we should change the check on R version. If someone had R 3.5 and it works then that's fine. Just that the message suggests you use R 3.6

**Comment by dannyparsons** [comment]
actually I think its good to change this now, its more work to keep maintaining R 3.5 and R 3.6 has been available for some time now, so ignore last message.

File: instat/clsRLink.vb [hunk 1]
```diff
@@ -81,7 +81,7 @@ Public Class RLink
             clsEngine.Initialize()
         Catch ex As Exception
             ' Modified message since currently not working for R 3.5.0
-            MsgBox(ex.Message & Environment.NewLine & "Could not establish connection to R." & vbNewLine & "R-Instat requires version " & strRVersionMajorRequired & "." & strRVersionMinorRequired & ".4 of R." & vbNewLine & "Note that R-Instat does not currently work with R 3.5.0 or above. Try reruning the installation to install R 3.4.4 or download R 3.4.4 from https://cran.r-project.org/bin/windows/base/old/3.4.4/ and restart R-Instat." & vbNewLine & ex.Message, MsgBoxStyle.Critical, "Cannot initialise R connection.")
+            MsgBox(ex.Message & Environment.NewLine & "Could not establish connection to R." & vbNewLine & "R-Instat requires version " & strRVersionMajorRequired & "." & strRVersionMinorRequired & ".0 of R." & vbNewLine & "Note that R-Instat does not currently work with R 3.5.0 or above. Try reruning the installation to install R 3.6.0 or download R 3.6.0 from https://cran.r-project.org/bin/windows/base/old/3.4.4/ and restart R-Instat." & vbNewLine & ex.Message, MsgBoxStyle.Critical, "Cannot initialise R connection.")
```
Comment (maxwellfundi) [inline instat/clsRLink.vb]: The link there needs to be updated to -https://cran.r-project.org/bin/windows/base/old/3.6.0/

Comment (stevenndungu) [reply instat/clsRLink.vb]: Yes for sure. Effecting the change

File: instat/clsRLink.vb [hunk 2]
```diff
@@ -80,8 +80,8 @@ Public Class RLink
...
-            MsgBox(ex.Message & Environment.NewLine & "Could not establish connection to R." & vbNewLine & "R-Instat requires version " & strRVersionMajorRequired & "." & strRVersionMinorRequired & ".4 of R." & vbNewLine & "Note that R-Instat does not currently work with R 3.5.0 or above. Try reruning the installation to install R 3.4.4 or download R 3.4.4 from https://cran.r-project.org/bin/windows/base/old/3.4.4/ and restart R-Instat." & vbNewLine & ex.Message, MsgBoxStyle.Critical, "Cannot initialise R connection.")
+            ' Modified message since currently not working for R 3.5.0 rather 3.6.0
+            MsgBox(ex.Message & Environment.NewLine & "Could not establish connection to R." & vbNewLine & "R-Instat requires version " & strRVersionMajorRequired & "." & strRVersionMinorRequired & ".0 of R." & vbNewLine & "Note that R-Instat does not currently work with R 3.5.0 or above. Try reruning the installation to install R 3.6.0 or download R 3.6.0 from https://cran.r-project.org/bin/windows/base/old/3.6.0/ and restart R-Instat." & vbNewLine & ex.Message, MsgBoxStyle.Critical, "Cannot initialise R connection.")
```
Comment (maxwellfundi) [inline instat/clsRLink.vb]: This message is not quite correct... - `"Note that R-Instat does not currently work with R 3.5.0 or above`
Should be the and below.

**Comment by stevenndungu** [comment]
@maxwellfundi have look at the changes. If they are ok now.

File: instat/clsRLink.vb (hunk 1 above)
Comment (maxwellfundi) [reply instat/clsRLink.vb]: Looks good @stevenndungu

**Pull request merged**